packages/frontend/src/components/ui/footer.tsx

```diff
@@ -5,13 +5,18 @@ import { Typography } from "@carrot-kpi/ui";
 
 const FooterLinks = ({ title, links }: FooterLink) => (
     <ul className="w-40">
-        <Typography className={{ root: "text-white" }} uppercase>
+        <Typography
+            data-testid={`${title}-text`}
```

Comment:
Same here, I think as a rule we could prefix the test ids with the component name, here it would be `footer-...`.

Reply:
agreed